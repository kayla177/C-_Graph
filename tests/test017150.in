ENTITY VEL3P6W0 name_VEL3 author
ENTITY CZ73MGTM name_CZ73 conference
ENTITY WIGZAO28 name_WIGZ paper
ENTITY REJ604LG name_REJ6 author
ENTITY QY4KZBO7 name_QY4K author
ENTITY 97IKPYBQ name_97IK institution
ENTITY UD7XIAFP name_UD7X journal
ENTITY 41YPHP1H name_41YP paper
ENTITY 2SA65DGH name_2SA6 paper
ENTITY YUBRTXM9 name_YUBR institution
RELATIONSHIP REJ604LG authored_by YUBRTXM9 2.3
RELATIONSHIP CZ73MGTM collaborated_with VEL3P6W0 9.4
RELATIONSHIP WIGZAO28 published_in 2SA65DGH 5.2
RELATIONSHIP REJ604LG collaborated_with CZ73MGTM 10.0
RELATIONSHIP VEL3P6W0 published_in 41YPHP1H 8.7
RELATIONSHIP UD7XIAFP published_in YUBRTXM9 4.8
RELATIONSHIP QY4KZBO7 collaborated_with REJ604LG 4.9
RELATIONSHIP 97IKPYBQ published_in VEL3P6W0 6.5
RELATIONSHIP 2SA65DGH cited_by REJ604LG 9.7
RELATIONSHIP 97IKPYBQ cited_by 41YPHP1H 5.7
RELATIONSHIP CZ73MGTM published_in 41YPHP1H 7.9
RELATIONSHIP QY4KZBO7 cited_by CZ73MGTM 0.4
RELATIONSHIP 97IKPYBQ authored_by QY4KZBO7 3.7
RELATIONSHIP QY4KZBO7 published_in WIGZAO28 5.3
RELATIONSHIP CZ73MGTM collaborated_with UD7XIAFP 7.5
PRINT YUBRTXM9
DELETE 41YPHP1H
HIGHEST
DELETE 97IKPYBQ
PATH CZ73MGTM VEL3P6W0
DELETE VEL3P6W0
HIGHEST
PATH CZ73MGTM YUBRTXM9
HIGHEST
PATH QY4KZBO7 WIGZAO28
EXIT
