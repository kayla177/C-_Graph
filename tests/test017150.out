success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
REJ604LG UD7XIAFP
success
97IKPYBQ 2SA65DGH 35.6
success
CZ73MGTM VEL3P6W0 9.4
success
WIGZAO28 UD7XIAFP 32.4
CZ73MGTM UD7XIAFP YUBRTXM9 12.3
WIGZAO28 UD7XIAFP 32.4
QY4KZBO7 WIGZAO28 5.3
